dashboard/src/components/workspace/WorkspaceInvoices.tsx

```diff
@@ -6,37 +6,37 @@ import { triggerToast } from '@/utils/toast';
 import { useState } from 'react';
 
 export function WorkspaceInvoices() {
+  const { currentWorkspace } = useCurrentWorkspaceAndApplication();
   const [loading, setLoading] = useState(false);
 
-  const { currentWorkspace } = useCurrentWorkspaceAndApplication();
+  const handleViewInvoices = async () => {
+    setLoading(true);
+    const { res, error } = await nhost.functions.call('/stripe-create-portal', {
```

Comment:
Did you try passing generics to `call`? The build is failing ATM because of a type error